src/Control/Monad/Trans/State/Persistent.hs

```diff
@@ -1,16 +1,17 @@
-{-# LANGUAGE PackageImports, ScopedTypeVariables #-}
+{-# LANGUAGE PackageImports      #-}
+{-# LANGUAGE ScopedTypeVariables #-}
 -- | In which the state of a State monad is persisted to disk.
 module Control.Monad.Trans.State.Persistent where
 
-import Control.Applicative
-import Control.Monad
-import Control.Monad.IO.Class
-import "mtl" Control.Monad.Trans
-import Control.Monad.Trans.Maybe
-import Control.Monad.Trans.State
-import Data.Functor.Identity
-import System.Directory
-import System.IO
+import           Control.Applicative
+import           Control.Monad
+import           Control.Monad.IO.Class
+import           "mtl" Control.Monad.Trans
```

Comment:
Hmm, this is a corner case. The reason I like the gap before the module names is so that identical `Control.` prefixes become aligned, but now the `"mtl"` unaligns it. Maybe the `"mtl"` could be moved into the gap? But then what happens if I need both `qualified` and the package name?